oshdb-util/src/main/java/org/heigit/bigspatialdata/oshdb/util/time/ISODateTimeParser.java

```diff
@@ -18,8 +18,11 @@
 
   /**
    * Converts an ISO 8601 Date or combined Date-Time String into a UTC based ZonedDateTime Object.
+   * <br>
```

Comment:
would it be better to use `<p>` here, since these should semantically be different paragraphs, right?